src/DisputeManager.ts

```diff
@@ -186,7 +197,51 @@ function updateAppeal(disputeId: BigInt, roundNumber: BigInt, event: EthereumEve
   if (appeal.opposedRuling.gt(BigInt.fromI32(0))) {
     appeal.confirmedAt = event.block.timestamp
   }
+
   appeal.save()
+  createAppealFeesForDeposits(disputeId, roundNumber, appeal as Appeal, event)
```

Comment:
Wonder if this should actually be called from `handleAppealDepositSettled()` since `updateAppeal()` is called when the round is appealed or appeal confirmed only.

Reply:
That's totally true, thx @fabriziovigevani !